### PR TITLE
Use `brew bundle` instead of custom functions

### PR DESCRIPTION
**Pull request by croaky**

### CODE REVIEW

**Comment by croaky** [comment]
@mikemcquaid @gohanlon @AlJohri How's this look?

**Comment by croaky** [comment]
@gabebw Your thoughts as well?

File: README.md [hunk 1]
```diff
@@ -129,13 +129,11 @@ For example:
 ```sh
 #!/bin/sh
 
-brew_install_or_upgrade 'go'
-brew_install_or_upgrade 'ngrok'
-brew_install_or_upgrade 'watch'
-
-fancy_echo "Cleaning up old Homebrew formulae ..."
-brew cleanup
-brew cask cleanup
```
Comment (mikemcquaid) [inline README.md]: You may still want the `cleanup`s; `brew bundle` won't do that for you.

File: mac [hunk 2]
```diff
@@ -137,45 +94,45 @@ if brew list | grep -Fq brew-cask; then
...
-brew_install_or_upgrade 'rbenv'
-brew_install_or_upgrade 'ruby-build'
-
-fancy_echo "Updating databases ..."
-brew_install_or_upgrade 'postgres'
-brew_install_or_upgrade 'redis'
-brew_launchctl_restart 'postgresql'
-brew_launchctl_restart 'redis'
+brew bundle --file=- <<EOF
+tap "thoughtbot/formulae"
```
Comment (mikemcquaid) [inline mac]: The extreme pedant in me winced at the use of double-quotes in this `Brewfile` and single-quotes in the other :wink:

**Comment by mikemcquaid** [comment]
Some minor comments otherwise: LGTM :+1:

File: mac (hunk 2 above)
Comment (gabebw) [inline mac]: I agree, we should use double quotes everywhere.

File: README.md [hunk 3]
```diff
@@ -129,13 +129,11 @@ For example:
...
+brew bundle --file=- <<EOF
```
Comment (gabebw) [inline README.md]: Isn't that `--file=-` just fancy-pants!

Comment (mikemcquaid) [inline README.md]: :heart_eyes:

**Pull request merged**